### PR TITLE
Milestone_2: Query And Update Animals Table

### PR DESCRIPTION
**Pull request by tresorsawasawa**
Below is the list of tasks I completed in this PR:
- [ ] Added a column `species` of type `string` to the **animals** table and modified the `schema.sql` file.

- [ ] Inserted the following data:
    - Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
    - Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
    - Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
    - Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
    - Animal: His name is Boarman. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
    - Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
    - Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.
    - Added these insertions to the `data.sql` file.

- [ ] Inside a transaction, updated the animals table by setting the species column to unspecified. Then roll back the change and went back to the state before the transaction.
- [ ] Inside a transaction:
    - Update the `animals` table by setting the species column to `Digimon` for all animals that have a name ending on `mon`.
    - Update the `animals` table by setting the species column to `Pokemon` for all animals that don't have species already set.
    - Commit the transaction.
- [ ] Inside a transaction:
    - Delete all animals born after Jan 1st, 2022.
    - Create a savepoint for the transaction.
    - Update all animals' weight to be their weight multiplied by -1.
    - Rollback to the savepoint
    - Update all animals' weights that are negative to be their weight multiplied by -1.
    - Commit transaction

- [ ] Write  the following queries:
    - Added the command to find the number of animals.
    - Added the command to find animals that have never tried to escape.
    - Added the command to find the average weight of animals.
    - Added the command to find who escapes the most, neutered or not neutered animals.
    - Added the command to find the minimum and maximum weight of each type of animal.
    - Added the command to find the average number of escape attempts per animal type of those born between 1990 and 2000.
    - Added these queries to the `queries.sql` file.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals(
   date_of_birth date,
   escape_attemps INT NOT NULL,
   neutered boolean,
-  weight_kg decimal(5, 2)
+  weight_kg decimal(5, 2),
+  species varchar(50)
```
Comment (andersonlebon) [inline schema.sql]: - [ ] Kindly note that regarding the requirement you should not directly add the `species` column in the animal table instead you should add it by altering the table. hint  :point_down: 

```sql
   ALTER TABLE animals ADD species varchar(255);
```

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @tresorsawasawa 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (tresorsawasawa) [reply schema.sql]: Thank You for the review

**Review by andersonlebon — APPROVED** [review]
Hi @tresorsawasawa,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Good README file :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by tresorsawasawa** [comment]
Hey @andersonlebon,

Thank You for the review :pray: 
I really appreciate :+1: :+1:

**Pull request merged**